Apps/APN-UIKit/APN UIKit/APN UIKit/SceneDelegate.swift

```diff
@@ -15,47 +16,58 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         // Called as the scene is being released by the system.
         // This occurs shortly after the scene enters the background, or when its session is discarded.
         // Release any resources associated with this scene that can be re-created the next time the scene connects.
         // The scene may re-connect later, as its session was not necessarily discarded (see `application:didDiscardSceneSessions` instead).
     }
-
+    
     func sceneDidBecomeActive(_ scene: UIScene) {
         // Called when the scene has moved from an inactive state to an active state.
         // Use this method to restart any tasks that were paused (or not yet started) when the scene was inactive.
     }
-
+    
     func sceneWillResignActive(_ scene: UIScene) {
         // Called when the scene will move from an active state to an inactive state.
         // This may occur due to temporary interruptions (ex. an incoming phone call).
     }
-
+    
     func sceneWillEnterForeground(_ scene: UIScene) {
         // Called as the scene transitions from the background to the foreground.
         // Use this method to undo the changes made on entering the background.
     }
-
+    
     func sceneDidEnterBackground(_ scene: UIScene) {
         // Called as the scene transitions from the foreground to the background.
         // Use this method to save data, release shared resources, and store enough scene-specific state information
         // to restore the scene back to its current state.
     }
-
-
+    
+    
     func setVisibleWindow() {
-            // If previous user is not a guest login and credentials were used to login into the app
+        // If previous user is not a guest login and credentials were used to login into the app
         if let email = storage.userEmailId, !email.isEmpty, let name = storage.userName, !name.isEmpty {
-                let navigationController = UINavigationController(rootViewController: DashboardViewController
-                    .newInstance())
-                window?.rootViewController = navigationController
-            } else {
-                let navigationController = UINavigationController(rootViewController: LoginViewController.newInstance())
-                window?.rootViewController = navigationController
-            }
-            window?.makeKeyAndVisible()
+            let navigationController = UINavigationController(rootViewController: DashboardViewController
+                .newInstance())
+            window?.rootViewController = navigationController
+        } else {
+            let navigationController = UINavigationController(rootViewController: LoginViewController.newInstance())
+            window?.rootViewController = navigationController
         }
+        window?.makeKeyAndVisible()
+    }
+    
+    func scene(_ scene: UIScene, continue userActivity: NSUserActivity) {
+
+    }
+    
+    func scene(_ scene: UIScene, openURLContexts URLContexts: Set<UIOpenURLContext>) {
```

Comment:
Since the source code of this sample app will be viewed by customers as examples of how to use our SDK, I suggest adding comments to this PR on how a customer could setup app scheme deep links in their own app.